mmengine/hooks/checkpoint_hook.py

```diff
@@ -412,6 +413,8 @@ def _save_best_checkpoint(self, runner, metrics) -> None:
                     'is removed')
 
             best_ckpt_name = f'best_{key_indicator}_{ckpt_filename}'
+            # Replace illegal characters for filename with `_`
+            best_ckpt_name = re.sub('(\\\\|/)', '_', best_ckpt_name)
```

Comment:
`best_ckpt_name` does not  contain `\\` so using `best_ckpt_name = best_ckpt_name.replace('/', '_')` will be more efficient.